### PR TITLE
Add cuffquant and cuffnorm to SDK

### PR DESCRIPTION
**Pull request by JenkoB**

### CODE REVIEW

File: resdk/utils/resolwe.py [hunk 1]
```diff
@@ -10,7 +10,46 @@ class ResolweUtilsMixin(object):
     def run_bamplot(self, bam, genome, input_gff=None, input_region=None, stretch_input=None,
                     color=None, sense=None, extension=None, rpm=None, yscale=None, names=None,
                     plot=None, title=None, scale=None, bed=None, multi_page=None):
-        """Run bamplot."""
+        """Run ``bamplot`` on the list of bam files.
+
+        This method runs `bamplot`_ with bams, genome and gff or region
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: Include link to `bamplot` process. You can leave link to bamplot tool, but make clear that this function runs process on resolwe-bio which wrapps original bamplot.

File: resdk/utils/resolwe.py [hunk 2]
```diff
@@ -10,7 +10,46 @@ class ResolweUtilsMixin(object):
...
+        be auto-wrapped in list, if it is not already.
+
+        .. _bamplot:
+            https://pypi.python.org/pypi/bamplot/0.9.0
+
+        :param list bam: subset of bam files to run process on
+        :param str genome: Genome used in the process (options are HG18,
+            HG19, MM9 and MM10)
+        :param input_gff: id of annotation file is given
+        :type input_gff: int or `~resdk.resources.data.Data
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: Missing backtick at the end

File: resdk/utils/resolwe.py [hunk 3]
```diff
@@ -10,7 +10,46 @@ class ResolweUtilsMixin(object):
...
+        :param str input_region: enter a genomic region
+        :param int stretch_input: stretch the input regions to a minimum
+            length
+        :param str color: enter a colon separated list of colors
+        :param str sense: map to forward, reverse or'both strand,
+            default maps to both
+        :param int extension: extends reads by n bp, edfault value is
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: `edfault` -> `default`

File: resdk/utils/resolwe.py [hunk 4]
```diff
@@ -10,7 +10,46 @@ class ResolweUtilsMixin(object):
...
+             200bp
+        :param bool rpm: normalizes density to reads per million (rpm),
+            default is False
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: Add double backticks around default values (in all docstrings).

File: resdk/utils/resolwe.py [hunk 5]
```diff
@@ -76,3 +115,46 @@ def run_bamplot(self, bam, genome, input_gff=None, input_region=None, stretch_in
...
         return bamplot
+
+    def run_cuffnorm(self, cuffquant, replicates, annotation, labels, useERCC=None, threads=None):
+        """Run Cuffnorm_ for selected cuffquats.
+
+         This method runs `Cuffnorm`_ process with ``cuffquant``
+         ``replicates``, ``annotation``, ``labels``, ``useERCC`` and
+        ``threads`` parameters specified in arguments.
+
+        .. _Cuffnorm:
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: Same as above: include link to the process.

File: resdk/utils/resolwe.py [hunk 6]
```diff
@@ -76,3 +115,46 @@ def run_bamplot(self, bam, genome, input_gff=None, input_region=None, stretch_in
         bamplot = self.get_or_run(slug='bamplot', input=inputs)
 
         return bamplot
+
+    def run_cuffnorm(self, cuffquant, replicates, annotation, labels, useERCC=None, threads=None):
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: Change `useERCC` to `use_ercc` to follow python naming convention.

File: resdk/utils/resolwe.py [hunk 7]
```diff
@@ -76,3 +115,46 @@ def run_bamplot(self, bam, genome, input_gff=None, input_region=None, stretch_in
...
+         ``replicates``, ``annotation``, ``labels``, ``useERCC`` and
+        ``threads`` parameters specified in arguments.
+
+        .. _Cuffnorm:
+            http://cole-trapnell-lab.github.io/cufflinks/cuffnorm/
+
+        :param list cuffquant: subset of cuffquant files to run process
+            on
+        :param list replicates: list where sample groups and/or sample
+            replicates are defined.
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: Omit dot at the end

File: resdk/utils/resolwe.py [hunk 8]
```diff
@@ -76,3 +115,46 @@ def run_bamplot(self, bam, genome, input_gff=None, input_region=None, stretch_in
...
+        :param annotation: id of annotation file is given
+        :type annotation: int or `~resdk.resources.data.Data`
+        :param list labels: labels for easch sample group are defined
```
Comment (dblenkus) [inline resdk/utils/resolwe.py]: `easch` -> `each`

File: resdk/utils/sample.py [hunk 9]
```diff
@@ -140,3 +145,25 @@ def run_rose2(self, use_background=True, background_slug='', genome='HG19', tss=
...
 
         return results
+
+    def run_cuffquant(self, gff):
+        """Run Cuffquant_ for selected cuffquats.
+
+        This method runs `Cuffquant`_ process with ``annotation``
+        specified in arguments.
+
+         .. _Cuffquant:
```
Comment (dblenkus) [inline resdk/utils/sample.py]: Link to the process.

Comment (JenkoB) [inline resdk/utils/sample.py]: resolwed

File: resdk/utils/resolwe.py (hunk 5 above)
Comment (JenkoB) [inline resdk/utils/resolwe.py]: resolwed

File: resdk/utils/resolwe.py (hunk 1 above)
Comment (JenkoB) [inline resdk/utils/resolwe.py]: resolwed

**Pull request merged**